### DISCUSSION TITLE
TCMPR output - A/B DEPTH

### DISCUSSION THREAD

**Discussion by drmohanthota** [dtcenter/METplus#1637] [Statistical Computation] [answered]
Hi all,
I have one question regarding the TCMPR output...
There is a column in the TCMPR output named A/BDEPTH, when I looked in the user doc, it was mentioned it is the depth of the system and categorised as D-deep, S-shallow, M-medium and X-unknown
How to interpret this particular variable , does this variable indicates the vertical extent of the system? or it is the depth from ocean surface. I understand that it maybe using the central pressure. How does it computes the depth of the system and categorise?

**Comment by georgemccabe** [comment, marked as answer]
Hi @drmohanthota, I was informed by a colleague that the depth is not something that MET computes. It passed through the value it finds from the input and is often left blank in the ATCF files. I skimmed the [documentation for the GFDL Tracker](https://dtcenter.org/sites/default/files/community-code/gfdl/standalone_tracker_UG_v3.9a.pdf) but I did not find any mention of this field. @KathrynNewman or @mrinalbiswas , do you have any insight on this subject?

**Reply by mrinalbiswas** [reply]
Hi @drmohanthota @georgemccabe, I do not have an answer to the question. I have seen B decks and CARQ in A decks have the info but have rarely seen model A-decks specifying it. I have asked the GFDL tracker developer if he has any insights. Once I have more info, I will provide you an update.

**Reply by georgemccabe** [reply]
Hi @mrinalbiswas, I am just following up on this issue. Do you have an update?

**Reply by georgemccabe** [reply]
Hi @drmohanthota, I am sorry for the delay. Since these values are not being computed by the MET tools, we are not able to provide a clear answer. You could try to reach out to the developers of the GFDL Tracker to see if they can provide more information.
